storage/src/journal/variable.rs

```diff
@@ -240,21 +240,22 @@ impl<E: Storage + Metrics, V: Codec> Journal<E, V> {
         let size = u32::from_be_bytes(size.as_ref().try_into().unwrap());
```

Comment:
nit: multiple cases to `usize`, we should just do `as usize` at the end of this line?